app/commands/v2/put_content.rb

```diff
@@ -214,6 +202,10 @@ def content_id
         payload.fetch(:content_id)
       end
 
+      def document_type
+        content_item_attributes_from_payload[:document_type]
```

Comment:
Sorry to be a pain, could we swap this for just `payload[:document_type]` - just keeps it consistent with similar methods